app/controllers/reports/regions_controller.rb

```diff
@@ -33,18 +33,16 @@ def show
         hsh[facility.name] = Reports::RegionService.new(region: facility,
                                                         period: @period).call
       }
-    else
-      @show_current_period = false
-      @dashboard_analytics = @region.dashboard_analytics(period: :month,
-                                                         prev_periods: 6,
-                                                         include_current_period: false)
```

Comment:
@claudiovallejo We don't wanna show this info on the overview anymore, I guess?

Reply:
@harimohanraj89 that's right — I removed this query because we removed the new registrations table from Facility Reports "Overview"